crates/moon/tests/test_cases/mod.rs

```diff
@@ -5296,11 +5304,17 @@ fn test_moon_check_json_output() {
         "#]],
         );
         check(
-            &get_stdout_with_args_and_replace_dir(&dir, ["check", "--output-json"]),
+            &get_stderr_on_success_with_args_and_replace_dir(
+                &dir,
+                ["check", "--output-json", "-q"],
+            ),
+            expect![""],
+        );
+        check(
+            &get_stderr_on_success_with_args_and_replace_dir(&dir, ["check", "--output-json"]),
             expect![[r#"
-            {"$message_type":"diagnostic","level":"warning","loc":{"path":"$ROOT/main/main.mbt","start":{"line":3,"col":3,"offset":27},"end":{"line":3,"col":10,"offset":34}},"message":"Warning (Alert alert_2): alert_2","error_code":2000}
```

Comment:
should keep

Reply:
I suppose this should be in stdout in this case (instead of also migrated to stderr), in order to keep CLI, language server, etc. stable.